services/smtp.py

```diff
@@ -26,8 +27,13 @@ def plugin(srv, item):
     msg['From']         = sender
     msg['X-Mailer']     = srv.SCRIPTNAME
 
+    if not smtp_addresses:
+        srv.logging.warn("Skipped sending SMTP notification to %s, "
+                         "no addresses configured: %s" % (item.target, smtp_addresses))
```

Comment:
Sure, the second parameter will be None or an empty list. Thanks for spotting this!